src/Api/Controllers/CollectionsController.cs

```diff
@@ -248,17 +272,6 @@ private async Task<Collection> GetCollectionAsync(Guid id, Guid orgId)
         return collection;
     }
 
-
-    private async Task<bool> CanCreateCollection(Guid orgId, Guid collectionId)
-    {
-        if (collectionId != default)
-        {
-            return false;
-        }
-
-        return await _currentContext.CreateNewCollections(orgId);
-    }
-
     private async Task<bool> CanEditCollectionAsync(Guid orgId, Guid collectionId)
```

Comment:
This is duplicating logic in the authorization handler and should be moved there. I know that we're doing it incrementally, but this is a bit different because it's identical to the "manage access" check. It should at least be marked deprecated, but preferably just moved there so we don't have identical logic in 2 separate places.

Reply:
Will be handled in AC-1139